src/test/java/uk/gov/hmcts/reform/ucmc/handler/CreateClaimCallbackHandlerTest.java

```diff
@@ -0,0 +1,88 @@
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.jackson.JacksonAutoConfiguration;
+import org.springframework.boot.test.context.SpringBootTest;
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.ccd.client.model.SubmittedCallbackResponse;
+import uk.gov.hmcts.reform.ucmc.callback.CallbackParams;
+import uk.gov.hmcts.reform.ucmc.callback.CallbackType;
+import uk.gov.hmcts.reform.ucmc.model.ClaimValue;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.HashMap;
+import java.util.Map;
+
+import static java.lang.String.format;
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.ucmc.helpers.DateFormatHelper.DATE_TIME_AT;
+import static uk.gov.hmcts.reform.ucmc.helpers.DateFormatHelper.formatLocalDateTime;
+
+@SpringBootTest(classes = {CreateClaimCallbackHandler.class, JacksonAutoConfiguration.class})
+class CreateClaimCallbackHandlerTest extends BaseCallbackHandlerTest {
+
+    @Autowired
+    private CreateClaimCallbackHandler handler;
+
+    @Test
+    void shouldReturnExpectedErrorInMidEventWhenValuesAreInvalid() {
+        Map<String, Object> data = new HashMap<>();
+        data.put("claimValue", ClaimValue.builder().higherValue(1).lowerValue(10).build());
+
+        CallbackParams params = callbackParamsOf(data, CallbackType.MID);
+
+        AboutToStartOrSubmitCallbackResponse response = (AboutToStartOrSubmitCallbackResponse) handler.handle(params);
+
+        assertThat(response.getErrors())
+            .containsOnly("CONTENT TBC: Higher value must not be lower than the lower value.");
+    }
+
+    @Test
+    void shouldReturnNoErrorInMidEventWhenValuesAreValid() {
+        Map<String, Object> data = new HashMap<>();
+        data.put("claimValue", ClaimValue.builder().higherValue(10).lowerValue(1).build());
+
+        CallbackParams params = callbackParamsOf(data, CallbackType.MID);
+
+        AboutToStartOrSubmitCallbackResponse response = (AboutToStartOrSubmitCallbackResponse) handler.handle(params);
+
+        assertThat(response.getErrors()).isEmpty();
+    }
+
+    @Test
+    void shouldReturnNoErrorInMidEventWhenNoValues() {
+        CallbackParams params = callbackParamsOf(new HashMap<>(), CallbackType.MID);
+
+        AboutToStartOrSubmitCallbackResponse response = (AboutToStartOrSubmitCallbackResponse) handler.handle(params);
+
+        assertThat(response.getErrors()).isEmpty();
+    }
+
+    @Test
+    void shouldReturnExpectedSubmittedCallbackResponseObject() {
+        CallbackParams params = callbackParamsOf(new HashMap<>(), CallbackType.SUBMITTED);
+
+        SubmittedCallbackResponse response = (SubmittedCallbackResponse) handler.handle(params);
+
+        String documentLink = "https://www.google.com";
+        String responsePackLink = "https://formfinder.hmctsformfinder.justice.gov.uk/n9-eng.pdf";
```

Comment:
Would be nice to move this into `application.yml` so that it's configurable and not need a release in case this changes in future.

Reply:
happy to update in a separate ticket, but not related to the refactoring to use the callback framework :)